lib/spack/spack/test/concretize.py

```diff
@@ -1713,7 +1713,7 @@ def test_best_effort_coconcretize(self, specs, expected):
         (['hdf5+mpi', 'zmpi', 'mpich'], 'mpich', 2)
     ])
     def test_best_effort_coconcretize_preferences(
-            self, specs, expected_spec, occurances
+            self, specs, expected_spec, occurrences
```

Comment:
You need to fix also the fixture name at line 1698

Reply:
@alalazo Done. Please check.